src/openrct2/libopenrct2.vcxproj

```diff
@@ -58,7 +58,36 @@
   <ItemGroup>
     <ClInclude Include="..\..\resources\resource.h" />
     <ClInclude Include="**\*.h" />
-    <ClInclude Include="**\*.hpp" />
+    <ClInclude Include="actions\GuestSetNameAction.hpp" />
+    <ClInclude Include="actions\PlaceParkEntranceAction.hpp" />
+    <ClInclude Include="actions\PlacePeepSpawnAction.hpp" />
+    <ClInclude Include="actions\RideCreateAction.hpp" />
+    <ClInclude Include="actions\RideDemolishAction.hpp" />
+    <ClInclude Include="actions\RideSetName.hpp" />
+    <ClInclude Include="actions\RideSetStatus.hpp" />
+    <ClInclude Include="actions\SetParkEntranceFeeAction.hpp" />
+    <ClInclude Include="actions\StaffSetNameAction.hpp" />
+    <ClInclude Include="cmdline\CommandLine.hpp" />
+    <ClInclude Include="config\ConfigEnum.hpp" />
+    <ClInclude Include="config\IniReader.hpp" />
+    <ClInclude Include="config\IniWriter.hpp" />
+    <ClInclude Include="core\Collections.hpp" />
+    <ClInclude Include="core\Console.hpp" />
+    <ClInclude Include="core\Diagnostics.hpp" />
+    <ClInclude Include="core\FileIndex.hpp" />
+    <ClInclude Include="core\FileStream.hpp" />
+    <ClInclude Include="core\Guard.hpp" />
+    <ClInclude Include="core\IStream.hpp" />
+    <ClInclude Include="core\Json.hpp" />
+    <ClInclude Include="core\Math.hpp" />
+    <ClInclude Include="core\Memory.hpp" />
+    <ClInclude Include="core\Nullable.hpp" />
+    <ClInclude Include="core\Path.hpp" />
+    <ClInclude Include="core\Registration.hpp" />
+    <ClInclude Include="core\String.hpp" />
+    <ClInclude Include="core\StringBuilder.hpp" />
+    <ClInclude Include="core\StringReader.hpp" />
+    <ClInclude Include="core\Util.hpp" />
```

Comment:
Don't add new files from within the IDE, just create the file and reload the project.

Reply:
I plan to fix this once the code portion itself is fixed up.